src/rastervision/ml_tasks/semantic_segmentation.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+from rastervision.core.ml_task import MLTask
+from rastervision.core.box import Box
+
+
+class SemanticSegmentation(MLTask):
+
+    def get_train_windows(self, scene, options):
+        raster_source = scene.raster_source
+        meta = raster_source.image_dataset.meta
+        width = meta.get('width')
+        height = meta.get('height')
+
+        label_store = scene.ground_truth_label_store
+
+        chip_size = options.chip_size
+
+        windows = []
+        for i in range(100):  # XXX insensitive
+            xmin = np.random.randint(low=0, high=width-chip_size)
```

Comment:
Could use extent.make_random_square() instead.

Reply:
Thank you